pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -281,19 +283,117 @@ def __call__(self, time: float):
             Current simulation time
 
         """
+        asympt_uninf = []
```

Comment:
This name isn't clear - will this be a list of people? Surely all uninfected person are asymptomatic?

Reply:
Also maybe comment why we are storing this here